packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/CohortTable.js

```diff
@@ -128,6 +129,13 @@ export default class CohortTable extends React.PureComponent<Props> {
           render={(value, record) => record.state === fetchedState.DONE && formatDuration(value)}
         />
         <Column title="Spans" dataIndex="data.spans.length" key="address" />
+        <Column
+          title="Link"
+          className="CohortTable--linkColumn"
```

Comment:
You can use the util CSS class `ub-tx-center` for this.

Reply:
nice util, updated